share/completions/loginctl.fish

```diff
@@ -25,3 +25,23 @@ complete -c loginctl -x -n "not $seen $commands" -s P -d "Equivalent to --value
 complete -c loginctl -x -n "not $seen $commands" -l signal -s s -d "Which signal to send"
 complete -c loginctl -f -n "not $seen $commands" -l value -d "When showing properties, only print the value"
 complete -c loginctl -f -n "not $seen $commands" -l version -d "Show package version"
+
+
+function list_sessions
+    loginctl list-sessions --no-legend --no-pager --output=short | awk '{print $1"\t"$3" at "$5}'
```

Comment:
Two things:

First: These function names are too general. Fish functions have no namespaces, you've just clobbered any user function called "list_session". Call it `__fish_loginctl_sessions` (and similar for the others).

Secondly: This should use `string` instead to reduce our use of `awk` - which is an external tool with a few incompatible dialects in practical circulation. Something like

```fish
loginctl list-sessions --no-legend --no-pager --output=short | string replace -r '(\d+) \d+ (\S+) \S+ (\S+) .*' '$1\t$2 at $3'
```

For this, and similar for the others.